pkg/server/backing_image.go

```diff
@@ -62,7 +62,7 @@ func NewBackingImage(name, url, uuid, diskPathOnHost, diskPathInContainer string
 		URL:           url,
 		HostDirectory: hostDir,
 		WorkDirectory: workDir,
-		state:         StatePending,
+		state:         "",
```

Comment:
If the state is regarded as enum here, it's better to have a defined value instead of an empty string for the initial value.